cases/ajv.ts

```diff
@@ -0,0 +1,69 @@
+import Ajv from "ajv"
+import { Case } from './abstract';
+
+const ajv = new Ajv();
```

Comment:
We seem to have lots of lint errors:

```
 Error:    1:17  error    Replace `"ajv"` with `'ajv';`                                                                         prettier/prettier
Warning:    1:17  warning  Strings must use singlequote                                                                          quotes
Error:    6:8   error    Replace `"AjvTest"` with `'AjvTest'`                                                                  prettier/prettier
```

Please run lint fix :)

Reply:
Done, sorry, I am used to husky git hooks doing its thing before pushing :)